v03_pipeline/lib/tasks/trigger_hail_backend_reload.py

```diff
@@ -0,0 +1,36 @@
+import luigi
+import requests
+
+from luigi_pipeline.lib.hail_tasks import GCSorLocalTarget
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.model import Env
+from v03_pipeline.lib.paths import hail_backend_reload_success_for_run_path
+from v03_pipeline.lib.tasks.base.base_loading_pipeline_params import (
+    BaseLoadingPipelineParams,
+)
+
+logger = get_logger(__name__)
+
+
+@luigi.util.inherits(BaseLoadingPipelineParams)
+class TriggerHailBackendReload(luigi.Task):
+    run_id = luigi.Parameter()
+
+    def output(self):
```

Comment:
I bet we don't even need this, we can just re-run the task cause it's idempotent!

Reply:
that tracks, I added a `done` attribute to the task to track completion